source/configuring-ember/handling-deprecations.md

```diff
@@ -0,0 +1,154 @@
+A valueable attribute of the EmberJS framework is its use of [Semantic Versioning](http://semver.org/) to aid projects in keeping up with
+changes to the framework.  Before any function or API is removed it first goes through a deprecation period where the functionality is
+still supported, but usage of it generates a warning.  These warnings can pile up between major releases to a point where the amount of
+deprecation warnings that scroll through the console becomes overwhelming.
+
+![Deprecations Clouding up the Browser JavaScript Console](../../images/guides/configuring-ember/handling-deprecations/deprecations-in-console.png)
+
+Fortunately, Ember provides a way for projects to deal with deprecations in an organized and efficient manner.
+
+## Filtering Deprecations
+
+When your project has a lot of deprecations, you can start by filtering out deprecations that do not have to be addressed right away.  You
+can use [deprecation handlers](http://emberjs.com/api/classes/Ember.Debug.html#method_registerDeprecationHandler) API to check for what
+release a deprecated feature will be removed.  An example handler is shown below that filters out all deprecations that are not going away
+in release 2.0.0.
+
+
+``` app/initializers/main.js
+if (Ember.Debug && _.isFunction(Ember.Debug.registerDeprecationHandler)) {
+    Ember.Debug.registerDeprecationHandler((message, options, next) => {
+        if (options && options.until && options.until !== '2.0.0') {
+            return;
+        }
+        next(message, options);
+    });
+}
+```
+
+The deprecation handler API was released in Ember 2.1.  If you would like to leverage this API in a prior release of EmberJS you can install
+the [ember-debug-handlers-polyfill](http://emberobserver.com/addons/ember-debug-handlers-polyfill) addon into your Ember project.
+
+## Deprecation Workflow
+
+Once you've removed deprecations that you may not need to immediately address, you may still be left with many deprecations.  Also, your remaining
+deprecations may only occur and very specific scenarios that are not obvious.  How then should you go about finding and fixing these?  This
+is where the [ember-cli-deprecation-workflow](http://emberobserver.com/addons/ember-cli-deprecation-workflow) addon can be extremely helpful.
+
+Once installed, the deprecation workflow plugin works in 3 steps:
+
+### 1. Gather deprecations into one source
+
+The ember-cli-deprecation-workflow addon provides a command that will collect deprecations from your console and generate JavaScript code listing
+its findings.
+
+To collect deprecations, first run your in-browser test suite.  If your test suite isn't fully covering your app's functionality, you may also
```

Comment:
"... run your in-browser test suite _by starting your development server and navigating to http://localhost:4200/tests_."